utils/vim/syntax/swift.vim

```diff
@@ -49,27 +49,27 @@ syn keyword swiftKeyword
       \ while
 
 syn keyword swiftTypeDefinition class extension protocol struct typealias enum skipwhite nextgroup=swiftTypeName
-syn region swiftTypeAttributes start="\[" end="\]" skipwhite contained nextgroup=swiftTypeName
+syn region swiftTypeAttributes start="\[" end="\]" contained skipwhite nextgroup=swiftTypeName
 syn match swiftTypeName /\<[A-Za-z_][A-Za-z_0-9\.]*\>/ contained nextgroup=swiftTypeParameters
 
-syn region swiftTypeParameters start="<" end=">" skipwhite contained
+syn region swiftTypeParameters start="<" end=">" contained
 
 syn keyword swiftMutating mutating skipwhite nextgroup=swiftFuncDefinition
 syn keyword swiftFuncDefinition func skipwhite nextgroup=swiftFuncAttributes,swiftFuncName,swiftOperator
-syn region swiftFuncAttributes start="\[" end="\]" skipwhite contained nextgroup=swiftFuncName,swiftOperator
-syn match swiftFuncName /\<[A-Za-z_][A-Za-z_0-9]*\>/ skipwhite contained nextgroup=swiftTypeParameters
+syn region swiftFuncAttributes start="\[" end="\]" contained skipwhite nextgroup=swiftFuncName,swiftOperator
```

Comment:
The bracket syntax for attributes is very old and not used in current Swift.  Feel free to remove support for it.